walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/PinRequirement.kt

```diff
@@ -27,7 +27,7 @@ class PinRequirement(
     val salt: String? = null,
 
     internal var pin: String? = null
-): Requirement {
+) : Requirement {
```

Comment:
Nit: was this intentional?

Reply:
Yes, it is. Changed it as fix for code analysis failure.